Packs/FireEyeETP/Integrations/FireEyeETP/FireEyeETP_test.py

```diff
@@ -75,18 +77,20 @@ def test_fetch_incident_by_status_messages(mocker):
     Then:
         - Ensure one incident was fetched as expected
     """
+    iso_format = "%Y-%m-%dT%H:%M:%S.%f"
+    current_time = datetime.now().strftime(iso_format)
```

Comment:
use `@freeze_time` to prevent flakiness in unit tests, rather than relying on the timestamp at runtime